index.html

```diff
@@ -9,138 +9,336 @@
-        <img src="placeholder.png" alt="">
-      </figure>
-      
-      <section class="ellipse_20"></section>
-      
-      <article>
-        <h1 class="title_post">Multi-Post Stories</h1>
-      </article>
-  
-      <article>
-        <p>
-          A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. has been the 
-          industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
-        </p>
+      <hr>
+    </section>  
+    
+    <section id="frame_container">
+      <article class="frame">
+        <section class="languages">
+          <h3>Languages</h3>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </section>
+
+        <div class="back"></div>
+        
+        <section class="frameworks">
+          <h3>Frameworks</h3>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </section>
       </article>
-  
-      <article class="tags">
+
+      <article class="skills">
+        <h3>Skills</h3>
         <ul>
-          <li>css</li>
-          <li>html</li>
-          <li>javascript</li>
-          <li>Ruby</li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
         </ul>
       </article>
-  
-      <section class="button">See Projects</section>
     </section>
-    <!--  -->
-    <section class="pro_3">
-      <figure class="img_placeholder">
-        <img src="placeholder.png" alt="">
-      </figure>
-  
-      <article>
-        <h1 class="title_post">Multi-Post Stories</h1>
-      </article>
 
-      <section class="ellipse_21"></section>
+    <hr>
+    
+    <section id="social_mediaWork">
+      <section class="work_first">
+        <a href="#"><img src="Element_Right.png" alt=""></a>
+      </section>
 
-      <article>
-        <p>
-          A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. has been the 
-          industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
-        </p>
-      </article>
-  
-      <article class="tags">
+      <article class="social-media">
         <ul>
-          <li>css</li>
-          <li>html</li>
-          <li>javascript</li>
-          <li>Ruby</li>
+          <li><a href="#"><img src="one.png" alt=""></a></li>
+          <li><a href="#"><img src="Frame.png" alt=""></a></li>
+          <li><a href="#"><img src="three.png" alt=""></a></li>
+          <li><a href="#"><img src="four.png" alt=""></a></li>
+          <li><a href="#"><img src="five.png" alt=""></a></li>
         </ul>
       </article>
   
```

Comment:
- [ ] I would suggest you, please have the image bigger than the text content on the left. See the image for reference.
- [ ] Kindly make changes to all the sections.
### Your design
![your](https://user-images.githubusercontent.com/81727988/186727192-cc9e07b0-f6f8-4ce6-8441-fc2cfb98a45f.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/186727242-7644cdec-4ea9-49f4-bf3e-1b64f04957a3.png)